File: unical-support-interface/src/app/model/email-to-classify-dto.ts
import {EmailDto} from './email-dto';
import {SingleClassificationDto} from './classifier-result-dto';

export class EmailToClassifyDto extends EmailDto{
  explanation: String;
  singleClassifications: SingleClassificationDto[];

  constructor(data: any) {
    super(data);
    this.explanation = data.explanation;
    this.singleClassifications = data.singleClassifications;
  }
}

File: unical-support-2/src/main/java/unical_support/unicalsupport2/data/dto/email/EmailToClassifyDto.java
package unical_support.unicalsupport2.data.dto.email;

import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.List;

@Data
@EqualsAndHashCode(callSuper = true)
public class EmailToClassifyDto extends EmailDto {
    private String explanation;
    private List<SingleClassificationDto> singleClassifications;
}
